unreleased.md

```diff
@@ -1 +1,8 @@
 # Unreleased
+
+## Features
+
+- `DomainLayer` is added which provides a `DomainModel` object through layer
```

Comment:
It will be a beta feature, better to indicate this in release notes.

_This is just a note. Leave it as is for now, we'll revise this at the end of the work anyway._